health/health_config.c

```diff
@@ -23,6 +23,9 @@
 #define HEALTH_RECIPIENT_KEY "to"
 #define HEALTH_UNITS_KEY "units"
 #define HEALTH_INFO_KEY "info"
+#define HEALTH_CLASS_KEY "class"
```

Comment:
These `define` are duplicated definition with values defined inside `health/health.h`.

Reply:
Thanks! I've left them in `health_config.c` (which is where they are actually used), and removed them from `health.h', with https://github.com/netdata/netdata/pull/10961/commits/797f0c7bfc5836b3628d7f13e14861d0cd1e3343